contrib/guix/README.md

```diff
@@ -171,9 +171,9 @@ details.
   Override the space-separated list of platform triples for which to perform a
   bootstrappable build.
 
-  _(defaults to "x86\_64-linux-gnu arm-linux-gnueabihf aarch64-linux-gnu
-  riscv64-linux-gnu powerpc64-linux-gnu powerpc64le-linux-gnu
-  x86\_64-w64-mingw32 x86\_64-apple-darwin arm64-apple-darwin")_
+  _(defaults to "x86\_64-linux-gnu aarch64-linux-gnu arm-linux-gnueabihf
+  riscv64-linux-gnu i686-linux-gnu x86\_64-w64-mingw32 x86\_64-unknown-freebsd
+  x86\_64-apple-darwin aarch64-apple-darwin aarch64-linux-android")_
```

Comment:
https://github.com/monero-project/monero/blob/1179269478f9b615d4b550c38b9ba81c77901aec/contrib/guix/guix-build#L82-L91

Are these the correct default targets?

Reply:
Yep, nice catch.